Error installing on macOS

Node version: 10.4.0

```
../src/WebWorkerThreads.cc:205:3: error: reference to 'ArrayBufferAllocator' is ambiguous
  ArrayBufferAllocator a;
  ^
../src/ArrayBufferAllocator.h:3:7: note: candidate found by name lookup is 'ArrayBufferAllocator'
class ArrayBufferAllocator : public v8::ArrayBuffer::Allocator {
      ^
/Users/nicholas/.node-gyp/10.4.0/include/node/node.h:217:7: note: candidate found by name lookup is
      'node::ArrayBufferAllocator'
class ArrayBufferAllocator;
      ^
../src/WebWorkerThreads.cc:207:32: error: use of undeclared identifier 'a'
  cp.array_buffer_allocator = &a;
                               ^
```
Let me know if you need more information...

--- a/src/ArrayBufferAllocator.h
+++ b/src/ArrayBufferAllocator.h
@@ -1,5 +1,6 @@
 // from: https://developers.google.com/v8/get_started
 
+namespace WebWorkerThreads {
 class ArrayBufferAllocator : public v8::ArrayBuffer::Allocator {
  public:
   virtual void* Allocate(size_t length) {
@@ -9,3 +10,4 @@ class ArrayBufferAllocator : public v8::ArrayBuffer::Allocator {
   virtual void* AllocateUninitialized(size_t length) { return malloc(length); }
   virtual void Free(void* data, size_t) { free(data); }
 };
+}

--- a/src/WebWorkerThreads.cc
+++ b/src/WebWorkerThreads.cc
@@ -202,7 +202,7 @@ static void aThread (void* arg) {
 
 #if NODE_MODULE_VERSION > 45
   // ref: https://developers.google.com/v8/get_started
-  ArrayBufferAllocator a;
+  WebWorkerThreads::ArrayBufferAllocator a;
   v8::Isolate::CreateParams cp;
   cp.array_buffer_allocator = &a;
   thread->isolate= Isolate::New(cp);

